crowbar_framework/app/assets/stylesheets/content/header.scss

```diff
@@ -1,35 +1,21 @@
 .navbar-inverse {
   .navbar-brand {
-    padding: 4px 15px 0;
+    padding: 0 15px 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0 15px` instead of `0 15px 0`